wolfcrypt/src/asn.c

```diff
@@ -2759,8 +2778,20 @@ int ToTraditionalInline_ex(const byte* input, word32* inOutIdx, word32 sz,
     idx = idx - 1; /* reset idx after finding tag */
 
     if (tag == ASN_OBJECT_ID) {
+        #ifdef HAVE_ECC
+        ret = GetObjectId(input, &idx, &oidSum, oidCurveType, sz);
+        if (ret == 0) {
+            if ((ret = CheckCurve(oidSum)) < 0){
```

Comment:
This check should be happening in GetObjectId() now.
Please verify.

Reply:
Verify the return value from GetObjectId(). To retrieve curve id, call CheckCurve()